src/JeroenArsenal/JNA/fn_arsenal_container.sqf

```diff
@@ -321,6 +322,7 @@ switch _mode do {
 			_ctrlTab ctrlSetForegroundColor _color;
 		} forEach jnva_loadout;
 	};
+*/
```

Comment:
Looks like you've commented out the bracket which should match to the bracket at line 307.

Reply:
🙄